tests/Behat/features/element-editor.feature

```diff
@@ -0,0 +1,90 @@
+      And I fill in "<p>My sample content</p>" for the "HTML" HTML field
+      And I click "Create" in the "#Form_ItemEditForm_action_doSave" element
+      # Remove with 'And I click "Blocks Page" in the ".breadcrumbs-wrapper" element' once the ElementalArea refreshes,
+      # See https://github.com/dnadesign/silverstripe-elemental/issues/320
+      And I go to "/admin/pages/edit/show/6"
+      And I wait until I see the ".element-editor__element" element
+    Then I should see "My Sample Block"
+
+  Scenario: I can see the icon, title, and summary of each element
+    Given I wait until I see the ".element-editor__element" element
+    Then I should see a list of blocks
+      And I should see "My Sample Block" as the title for block 1
+
+  Scenario: I can add elements to the page
+    When I select "Content" from "elemental-editor_add-new-block-control_select-dropdown"
+      And I click "Add" in the ".elemental-editor__add-new-block-control" element
+      And I fill in "Additional Sample Block" for "Title"
+      And I fill in "<p>Additional sample content</p>" for the "HTML" HTML field
+      And I click "Create" in the "#Form_ItemEditForm_action_doSave" element
+    Then I should see an ".message" element
+      And I should see a "Saved content block" message
+
+    When I go to "/admin/pages/edit/show/6"
+      And I wait until I see the ".element-editor__element" element
+      And I wait 1 second
+    Then I should see "Additional Sample Block"
+
+  Scenario: I can edit a block
+    Given I wait until I see the ".element-editor__element" element
+    Then I should see block 1
+
+    Given I click on block 1
+    Then I should see "My Sample Block"
+      And the "HTML" HTML field should contain "My sample content"
+
+    Given I fill in "New Block Title" for "Title"
+      And I fill in "<p>New sample content</p>" for the "HTML" HTML field
+      And I click "Publish" in the "#Form_ItemEditForm_action_doPublish" element
+      And I go to "/admin/pages/edit/show/6"
+    Then I should see "New Block Title"
+    But I should not see "My Sample Block"
+
+  @modal
+  Scenario: I can delete a block
+    Given I select "Content" from "elemental-editor_add-new-block-control_select-dropdown"
+      And I click "Add" in the ".elemental-editor__add-new-block-control" element
+      And I wait 1 second
+      And I fill in "Second Sample Block" for "Title"
+      And I fill in "<p>Additional sample content</p>" for the "HTML" HTML field
+      And I click "Create" in the "#Form_ItemEditForm_action_doSave" element
+      And I go to "/admin/pages/edit/show/6"
+      And I wait until I see the ".element-editor__element" element
+      And I wait 1 second
+      And I should see "Second Sample Block"
+      And I should see block 1
+      And I should see the delete button for block 1
+
+    Then I click on the delete button for block 1
+      And I see the text "Are you sure you want to delete this block?" in the alert
```

Comment:
`Then I should see...`

Reply:
This is a predefined step I am utilising from the `BasicContext.php`:

```php
/**
     * @Given /^I see the text "([^"]+)" in the alert$/
     * @param string $expected
     */
    public function iSeeTheDialogText($expected)
    {
        $text = $this->getExpectedAlert()->getText();
        assertContains($expected, $text);
    }
```